### ISSUE TITLE
Request to add "mpl_toolkits" especially "mpl_toolkits.map" as pyodide package

### ISSUE THREAD

**Issue by cosmosanalytics** [pyodide/pyodide#2962]
Request to add "mpl_toolkits" especially "mpl_toolkits.map" as pyodide package. 

NetworkX and Matplotlib "mpl_toolkits.map" work nicely together (see https://tuangauss.github.io/projects/networkx_basemap/networkx_basemap.html  ) 

![image](https://user-images.githubusercontent.com/100080151/184357444-0e8405da-ac51-4e17-8a8d-5310d503a2c8.png)

**Comment by rth** [comment]
Yes, it would be nice to add. We should have all the requirements already including GEOS and pyproj.
